pkg/runner/runner.go

```diff
@@ -308,11 +305,11 @@ func migrate(cfg *pgclient.Config, appVersion pgmodel.VersionInfo, leaseLock *ut
 }
 
 func checkDependencies(cfg *pgclient.Config, appVersion pgmodel.VersionInfo) error {
```

Comment:
Should this also use the schemaLeaseLock connection for fate-sharing?

Reply:
it doesn't hurt, but it matters less: all the Client connections will perform their own checking, so this check is really just so we can print a nice error message in the event we have the wrong version; it's not a correctness hazard.

Reply:
it was easy so I changed it